purchases/src/main/kotlin/com/revenuecat/purchases/common/offlineentitlements/OfflineEntitlementsManager.kt

```diff
@@ -116,10 +117,12 @@ internal class OfflineEntitlementsManager(
     }
 
     // We disable offline entitlements in observer mode (finishTransactions = true) since it doesn't
-    // provide any value and simplifies operations in that mode.
+    // provide any value and simplifies operations in that mode. Also on test store, since we don't have a store
+    // to store purchases in the client.
     private fun isOfflineEntitlementsEnabled() = appConfig.finishTransactions &&
         appConfig.enableOfflineEntitlements &&
-        !appConfig.customEntitlementComputation
+        !appConfig.customEntitlementComputation &&
+        appConfig.store != Store.TEST_STORE
```

Comment:
Nitpick: On iOS using enums I'd do something like this (not sure if this is a thing in Kotlin), but it avoids using the Store cases throughout the codebase which could be kind of nice.
```
enum Store {
  case appStore
  case test
}

extension Store {
  var isOfflineEntitlementsSupported: Bool {
    switch self {
    case .appStore: return true
    case .test: return false
    }
  }
}

Reply:
> I like the cleanup! Will do so tomorrow, thanks!